### ISSUE TITLE
Source map core exception frames

### ISSUE THREAD

**Issue by mfikes** [mfikes/replete#4]

**Comment by swannodette** [comment]
Ah there's already an issue for this! Would be cool if the frames were links that allowed you to navigate the stack trace.

**Comment by mfikes** [comment]
@swannodette yes! If we end up with `IPrintStacktrace` (or something similar) for bootstrapped REPLs, it would probably be easy to hyperlink frames to their source.

**Comment by mfikes** [comment]
Wondering if JSC supports inline source maps (useful for bootstrap context)
